replication-api-impl/src/test/java/com/connexta/replication/api/impl/persistence/spring/SiteRepositoryIntegrationTest.java

```diff
@@ -162,9 +208,53 @@ public void testPojoPersistenceWhenTypeIsNotDefined() throws Exception {
         new SitePojo()
             .setVersion(SiteRepositoryIntegrationTest.VERSION)
             .setId(SiteRepositoryIntegrationTest.ID)
-            .setRemoteManaged(SiteRepositoryIntegrationTest.REMOTE_MANAGED)
             .setName(SiteRepositoryIntegrationTest.NAME)
-            .setUrl(SiteRepositoryIntegrationTest.URL);
+            .setDescription(SiteRepositoryIntegrationTest.DESCRIPTION)
+            .setUrl(SiteRepositoryIntegrationTest.URL)
+            .setType(SiteRepositoryIntegrationTest.TYPE)
+            .setKind(SiteRepositoryIntegrationTest.KIND)
+            .setPollingTimeout(SiteRepositoryIntegrationTest.POLLING_TIMEOUT)
+            .setParallelismFactor(SiteRepositoryIntegrationTest.PARALLELISM_FACTOR);
+    final SitePojo saved = repo.save(pojo);
+
+    Assert.assertThat(saved, Matchers.equalTo(pojo));
+    Assert.assertThat(
+        repo.findById(SiteRepositoryIntegrationTest.ID),
+        OptionalMatchers.isPresentAnd(Matchers.equalTo(pojo)));
+  }
+
+  @Test
+  public void testPojoPersistenceWhenKindIsNotDefined() throws Exception {
+    final SitePojo pojo =
+        new SitePojo()
+            .setVersion(SiteRepositoryIntegrationTest.VERSION)
+            .setId(SiteRepositoryIntegrationTest.ID)
+            .setName(SiteRepositoryIntegrationTest.NAME)
+            .setDescription(SiteRepositoryIntegrationTest.DESCRIPTION)
+            .setUrl(SiteRepositoryIntegrationTest.URL)
+            .setKind(SiteRepositoryIntegrationTest.KIND)
```

Comment:
❓ Should this be `setKind(null)` or test renamed `...TypeIsNotDefined`?

Reply:
cut & paste error. Fixing.